### PR TITLE
Remove sysvinit-tools as an RPM package dependency

### PR DESCRIPTION
**Pull request by rossmcdonald**
###### Required for all non-trivial PRs
- [x] Rebased/mergable
- [x] Tests pass
- [x] CHANGELOG.md updated

This PR reverts https://github.com/influxdata/influxdb/pull/6835, which added sysvinit-tools as an RPM dependency.

/cc @jsternberg

### CODE REVIEW

**Comment by jsternberg** [comment]
This breaks non-systemd systems. The reason why this was added was for systems using sysvinit.

**Comment by rossmcdonald** [comment]
@jsternberg Can you provide the error you see when installing on non-systemd systems? I believe we've always supported non-systemd systems, and I don't recall seeing any issues.

If we want to include it as a dependency, then we'll need to maintain multiple versions of the packages: one for systemd, and one for sysvinit.

File: scripts/init.sh [hunk 1]
```diff
@@ -130,104 +83,147 @@ function log_success_msg() {
...
-	if test -n "$PID" && kill -0 "$PID" &>/dev/null; then
-	    # If the status is SUCCESS then don't need to start again.
-            log_failure_msg "$NAME process is running"
-            exit 0 # Exit
+    # Check that the PID file exists, and check the actual status of process
+    if [ -f $PIDFILE ]; then
+	      PID="$(cat $PIDFILE)"
+	      if kill -0 "$PID" &>/dev/null; then
+	          # Process is already up
+            log_success_msg "$NAME process is already running"
```
Comment (jsternberg) [inline scripts/init.sh]: There's some really weird indentation here.

File: scripts/init.sh [hunk 2]
```diff
@@ -130,104 +83,147 @@ function log_success_msg() {
...
 
 function status() {
     # Check the status of the process.
-    if [ -e $PIDFILE ]; then
-	PID="$(pgrep -f $PIDFILE)"
-	if test -n "$PID" && test -d "/proc/$PID" &>/dev/null; then
-            log_success_msg "$NAME Process is running"
+    if [ -f $PIDFILE ]; then
+	      PID="$(cat $PIDFILE)"
+        if kill -0 $PID &>/dev/null; then
```
Comment (jsternberg) [inline scripts/init.sh]: Some more weird indentation.

**Comment by jsternberg** [comment]
Looks fine for the most part. There's a lot of weird indentation so double check for tabs/spaces and the proper indentation and otherwise I think this is good to go.

File: scripts/init.sh (hunk 1 above)
Comment (rossmcdonald) [inline scripts/init.sh]: All weird indentation should be fixed. Looks like my editor mixed in tabs and spaces for indentation.

**Comment by jsternberg** [comment]
Squash the commits and it LGTM on green. Have you tried installing it on both CentOS 6 and CentOS 7 to double check that it works?

**Comment by rossmcdonald** [comment]
@jsternberg Rebased, and tested on CentOS 5, 6, and 7, along with Ubuntu Trusty, Wily, and Xenial without issue.

**Pull request merged**